python/cuml/cuml/tests/test_qn.py

```diff
@@ -68,7 +68,7 @@ def test_qn(loss, dtype, penalty, l1_strength, l2_strength, fit_intercept):
         cuml_score = accuracy_score(y_test, y_pred)
 
         assert cuml_score > baseline_score
-        assert cuml_score >= 0.50
+        assert cuml_score >= 0.45
```

Comment:
```suggestion
```

We can totally remove the line, the assert above already checks against the baseline score

Reply:
Removed in https://github.com/rapidsai/cuml/pull/6636/commits/d34b34bf0536cc792ce99900fab710e2e231b4f1 .